src/shm/pool/raw.rs

```diff
@@ -94,6 +101,33 @@ impl RawPool {
         Ok(buffer)
     }
 
+    #[allow(clippy::too_many_arguments)]
+    pub(crate) fn create_buffer_raw(
```

Comment:
is this something other pool implementations (out of crate implementations especially) could use and justify making this function public?

Reply:
I think so, although depending on how the client API works out, it may not be needed as long as the `pool()` function exists to allow direct access to the interface.